### DISCUSSION TITLE
Add simple input dialog?

### DISCUSSION THREAD

**Discussion by jonas-s-s-s** [ookii-dialogs/ookii-dialogs-wpf#47] [Ideas]
I'm not sure if I missed anything, but from what I found out there is no dialog for user value input other than Credential Dialog?
It would be nice to add an 1 value input dialog with Label, textbox, and Yes/Cancel buttons. 
I mean something like this:
![pic](https://user-images.githubusercontent.com/13944761/116829780-1af33780-aba6-11eb-940c-f995c4cffaec.png)

There already is Microsoft.VisualBasic.Interaction.InputBox, however that one is not good at all, because it's outdated and doesn't allow you to check if user clicked the cancel button or not (it just returns empty string after clicking the cancel button).

**Comment by augustoproiete** [comment]
Hey @jonasx10j, indeed the WPF version of Ookii Dialogs doesn't have an input dialog like you mentioned.

The Windows Forms version of Ookii Dialogs, however, has an [Input Dialog](https://github.com/ookii-dialogs/ookii-dialogs-winforms#input-dialog): 

![image](https://user-images.githubusercontent.com/177608/130853209-32009498-ce1d-41f1-a5ae-b1fc5db83074.png)

We should implement a WPF version of this dialog here. Tracking via https://github.com/ookii-dialogs/ookii-dialogs-wpf/issues/52
